rita/src/rita_client/exit_manager/mod.rs

```diff
@@ -179,7 +179,14 @@ pub fn exit_setup_request(
     reg_details.email_code = code;
 
     let ident = ExitClientIdentity {
-        global: SETTING.get_identity(),
+        global: match SETTING.get_identity() {
```

Comment:
are these possible any more? or should we panic?

Reply:
Handled, went with `ok_or` as suggested by Michał